odkx-src/files/cloud_init_AWS.yml

```diff
@@ -36,7 +36,7 @@ runcmd:
   - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
   - add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
   - apt-get update -y
-  - apt-get install -y docker-ce docker-ce-cli containerd.io
+  - apt-get install -y -m docker-ce docker-ce-cli containerd.io
```

Comment:
Hello,
This looks good - I just have a small comment, not directly related to this PR, but it appears that docker can no longer (since yesterday!) be installed like that on Ubuntu 20.04 (Focal Fossa). According to [askubuntu the release file for Docker is no longer available](https://askubuntu.com/questions/1230189/how-to-install-docker-community-on-ubuntu-20-04-lts). On 20.04 it can either be installed using the Bionic repository or simply by "apt install docker.io". I am not sure of the reasons why nor the implications, but I will open a separate issue regarding that ...
Anyway - good work :-)